nginx_proxy/config.json

```diff
@@ -28,6 +31,9 @@
     "certfile": "str",
     "keyfile": "str",
     "hsts": "str",
+    "cloudflare": {
```

Comment:
I think a simple `cloudflare: bool` should be better

Reply:
I agree, simplified the config.json to "cloudflare": "bool" and default "cloudflare": false, updated run.sh to reflect the change and check if "$CLOUDFLARE" == "true"